recipes/gretel/meta.yaml

```diff
@@ -5,19 +5,19 @@ package:
   version: {{ version }}
 
 build:
-  number: 1
-  skip: True  # [not py27]
+  number: 0
+  noarch: python
 
 source:
   url: https://github.com/SamStudio8/gretel/archive/v{{ version }}.tar.gz
   sha256: 1b111e1c56944e8f712fd1aada12b450cb97e9a196437a6c374ebbe51b9209b1
 
 requirements:
   host:
-    - python
+    - python <3
```

Comment:
gretel works with python3, do I need the `<3` here?

Reply:
No, remove it